Keep perlpath from messing with files named configure or configure.in (such
as the ones included with libica).
--- openssl-0.9.7a/util/perlpath.pl	2003-06-09 23:06:36.000000000 -0400
+++ openssl-0.9.7a/util/perlpath.pl	2003-06-09 23:06:25.000000000 -0400
@@ -11,6 +11,7 @@
 
 sub wanted
 	{
+	return if /^configure/;
 	return unless /\.pl$/ || /^[Cc]onfigur/;
 
 	open(IN,"<$_") || die "unable to open $dir/$_:$!\n";
